src/lib/Libtpp/tpp_transport.c

```diff
@@ -980,8 +974,6 @@ tpp_post_cmd(int tfd, char cmd, tpp_packet_t *pkt)
 
```

Comment:
we don't need `tpp_unlock_rwlock(&cons_array_lock);` anymore at line 970 above

Also at line number 866, it will be safe to put `free_router(r);` under lock/unlock